roles/openshift_logging_curator/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
 openshift_logging_curator_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
-openshift_logging_curator_image_version: "{{ openshift_logging_image_version | default('latest') }}"
+openshift_logging_curator_image_version: "{{ openshift_logging_image_version | default(__openshift_logging_image_version) }}"
```

Comment:
It has been requested by the operations team that we have a default be a static value. We should not be using a 'private' variable as a default value. If this role is ever used separately from the `openshift_logging` role this default would fail.